programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -2018,6 +2025,10 @@ static void enroll_cb(ziti_config *cfg, int status, char *err, void *ctx) {
 }
 
 static void enroll(int argc, char *argv[]) {
+    uv_loop_t *l = uv_loop_new();
```

Comment:
`process_tunnel_commands` calls `enroll_ziti_async` which calls this function. is it appropriate to get the log level and log init here?

Reply:
also seems like `enroll` should take the loop?

Reply:
nevermind. it uses `ziti_enroll` instead. my bad! :) lgtm